composer/algorithms/low_precision_groupnorm/README.md

```diff
@@ -0,0 +1,92 @@
+# 🧁 Low Precision GroupNorm
+
+
+[\[How to Use\]](#how-to-use) - [\[Suggested Hyperparameters\]](#suggested-hyperparameters) - [\[Technical Details\]](#technical-details) - [\[Attribution\]](#attribution)
+
+ `Natural Language Processing`, `Math Equivalent`
+
+Low Precision GroupNorm forces `torch.nn.GroupNorm` modules to run in float16 or bfloat16 precision, improving utilization. This should not affect final convergence values, but in rare cases may cause loss spikes.
+
+
+## How to Use
+Low Precision GroupNorm wraps `torch.nn.GroupNorm`, forcing the module to run in a lower precision if you have autocast enabled. If you are running in Automatic Mixed Precision (`amp`) mode, Low Precision GroupNorm will run in `torch.float16`. If you are running in `bf16` mode, Low Precision GroupNorm will run in `torch.bfloat16`.
+
+This algorithm will have no effect if you are running in `fp32` or `fp16` mode.
```

Comment:
how do I know what mode I'm running in? Is it just based on the precision I passed to the trainer? What if I'm not using the trainer? Am I in one of these modes by default?

(think 1-2 sentences could clarify all of this)
EDIT: or maybe just indirecting to the discussion of how to set the precision later in this page